lib/reek/report/code_climate/code_climate_configuration.yml

```diff
@@ -662,3 +662,4 @@ UtilityFunction:
   remediation_points: 250_000
   content: |
     A _Utility Function_ is any instance method that has no dependency on the state of the instance.
+# TODO: add entry for SubclassedFromCoreClass
```

Comment:
Let's use the same remediation_points as for UtilityFunction above and add a short description. We can just c&p the description from the smell detector below, so : https://github.com/troessner/reek/pull/954/files#diff-8c155dc71177f640b21a8834d1aec4abR8